pre_processing/set_ecmwf.F90

```diff
@@ -291,150 +279,151 @@ real(dreal) function find_center_time(imager_geolocation, imager_time) &
-   case('06')
-      ecmwf_hour_hr = '600'
-   case('12')
-      ecmwf_hour_hr = '1200'
-   case('18')
-      ecmwf_hour_hr = '1800'
-   end select
-
-   hr_ext ='/'//trim(adjustl(yyyy))//'/'//trim(adjustl(mm))//'/'// &
-           trim(adjustl(dd))//'/ERA_Interim_an_'//trim(adjustl(yyyy))// &
-           trim(adjustl(mm))//trim(adjustl(dd))//'_'// &
-           trim(adjustl(ecmwf_hour_hr))//'+00_HR.grb'
-   ecmwf_hr_path_file = trim(adjustl(ecmwf_hr_path))//trim(adjustl(hr_ext))
-
-end subroutine build_ecmwf_HR_file_from_LR2
+subroutine determine_jasmin_filenames_era5(nwp_fnames, cyear, cmonth, cday, chour, idx)
+    use orac_ncdf_m
+    use preproc_constants_m
+    use preproc_structures_m
+
+    type(preproc_nwp_fnames_t), intent(inout) :: nwp_fnames
+    character(len=*), intent(in)              :: cyear, cmonth, cday, chour
+    integer, intent(in)                       :: idx
+    
+    character(len=path_length)                :: ml_dir, sfc_dir
+    
+    print*,cyear, ' ', cmonth, ' ', cday, ' ', chour
+    
+    ml_dir = trim(adjustl(nwp_fnames%nwp_path(idx)))//'an_ml/'// &
+             trim(adjustl(cyear))//'/'// &
+             trim(adjustl(cmonth))//'/'// &
+             trim(adjustl(cday))//'/ecmwf-era5_oper_an_ml_'// &
+             trim(adjustl(cyear))// &
+             trim(adjustl(cmonth))// &
+             trim(adjustl(cday))// &
+             trim(adjustl(chour))//'.'
+             
+    sfc_dir = trim(adjustl(nwp_fnames%nwp_path(idx)))//'an_sfc/'// &
+              trim(adjustl(cyear))//'/'// &
+              trim(adjustl(cmonth))//'/'// &
+              trim(adjustl(cday))//'/ecmwf-era5_oper_an_sfc_'// &
+              trim(adjustl(cyear))// &
+              trim(adjustl(cmonth))// &
+              trim(adjustl(cday))// &
+              trim(adjustl(chour))//'.'
+    
+    ! Set up surface variables   
+    nwp_fnames%ci_f(idx) = trim(adjustl(sfc_dir))//'ci.nc'
+    nwp_fnames%asn_f(idx) = trim(adjustl(sfc_dir))//'asn.nc'
+    nwp_fnames%tcwv_f(idx) = trim(adjustl(sfc_dir))//'tcwv.nc'
+    nwp_fnames%sd_f(idx) = trim(adjustl(sfc_dir))//'sd.nc'
+    nwp_fnames%u10_f(idx) = trim(adjustl(sfc_dir))//'10u.nc'
+    nwp_fnames%v10_f(idx) = trim(adjustl(sfc_dir))//'10v.nc'
+    nwp_fnames%t2_f(idx) = trim(adjustl(sfc_dir))//'2t.nc'
+    nwp_fnames%skt_f(idx) =  trim(adjustl(sfc_dir))//'skt.nc'
+    nwp_fnames%sstk_f(idx) = trim(adjustl(sfc_dir))//'sst.nc'
+    nwp_fnames%cape_f(idx) = trim(adjustl(sfc_dir))//'cape.nc'    
+      
+    ! Set up model level variables
+    nwp_fnames%q_f(idx) = trim(adjustl(ml_dir))//'q.nc'
+    nwp_fnames%t_f(idx) = trim(adjustl(ml_dir))//'t.nc'
+    nwp_fnames%o3_f(idx) = trim(adjustl(ml_dir))//'o3.nc'
+    nwp_fnames%lnsp_f(idx) = trim(adjustl(ml_dir))//'lnsp.nc'
+    nwp_fnames%u_f(idx) = trim(adjustl(ml_dir))//'u.nc'
+    nwp_fnames%v_f(idx) = trim(adjustl(ml_dir))//'v.nc'
+    nwp_fnames%z_f(idx) = trim(adjustl(ml_dir))//'z.nc'
+    
+    print*, trim(nwp_fnames%ci_f(idx))
```

Comment:
Aren't these debugging lines that should, at least, be in an `if (verbose)` block?

Reply:
Outdated commit so not relevant.